src/ol/events/event.js

```diff
@@ -55,6 +55,6 @@ ol.events.Event.stopPropagation = function(evt) {
 /**
  * @param {Event|ol.events.Event} evt Event
  */
-ol.events.Event.preventDefault = function(evt) {
+ol.events.Event.prototype.preventDefault = function(evt) {
```

Comment:
Same as above.  This is only used [one place](https://github.com/openlayers/ol3/blob/34d8d77344ee0b653770f065c593d4ab7b5d102b/src/ol/renderer/dom/dommaprenderer.js#L70-L72) in the library, but it cannot be renamed here without changing it there.  Again, if we are going to rename it, `ol.events.preventDefault` makes sense.  But again, it doesn't need to be accessible from every `ol.events.Event` instance.